include/tcpdf_fonts.php

```diff
@@ -2000,10 +2000,10 @@ public static function getUniord($uch) {
 	public static function UTF8StringToArray($str, $isunicode, &$currentfont) {
 		if ($isunicode) {
 			// requires PCRE unicode support turned on
-			$chars = TCPDF_STATIC::pregSplit('//','u', $str, -1, PREG_SPLIT_NO_EMPTY);
+			$chars = TCPDF_STATIC::pregSplit('//','u', $str ?? '', -1, PREG_SPLIT_NO_EMPTY);
```

Comment:
You can not use this syntax, is is not old PHP 5 compatible :/

Reply:
Ah! Not realizing that supporting PHP 5 was a requirement, fixed!